graph/db/kv_store.go

```diff
@@ -269,7 +269,7 @@ type channelMapKey struct {
 
 // getChannelMap loads all channel edge policies from the database and stores
 // them in a map.
-func (c *ChannelGraph) getChannelMap(edges kvdb.RBucket) (
+func (c *KVStore) getChannelMap(edges kvdb.RBucket) (
```

Comment:
nit: receiver could also be renamed to `s` (as in store).

Reply:
cool - since we've changed this PR to point to a side branch, I'll update this as first commit of next PR if this one gets no more comments to address, otherwise will update on next push here if more review comments 🫡

Reply:
or, if it is ok, perhaps as a final commit in this cache series since this receiver change will cause maximum rebase conflicts 🙃

Reply:
sgtm! :)